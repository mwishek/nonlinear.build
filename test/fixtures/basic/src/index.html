---
template: index.hbt
prismic:
  macaron:
    query: '[[:d = at(document.tags, ["Macaron"])] [:d = any(document.type, ["product"])]]'
    orderings: '[my.product.name]'
  jobOffers:
    query: '[[:d = any(document.type, ["job-offer"])]]'
---
