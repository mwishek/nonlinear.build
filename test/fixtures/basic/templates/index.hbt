<html>
<body>
<ul>
{{#each prismic.macaron.results}}
    <li>
        {{{data.name.html}}}
        {{{data.description.html}}}
    </li>
{{/each}}
</ul>
<ul>
{{#each prismic.jobOffers.results}}
    <li>
        {{{data.name.html}}}
        {{{data.location.html}}}
        {{{data.profile.html}}}
        {{{data.description.html}}}
    </li>
{{/each}}
</ul>
</body>
</html>
