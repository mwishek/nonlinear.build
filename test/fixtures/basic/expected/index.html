<html>
<body>
<ul>
    <li>
        <h1>Black &amp; White Macaron</h1>
        <p>If you feel like antique, experimented tastes are the way to go, then this macaron is perfect for you! A solid touch of dark chocolate, enveloped by a softening touch of vanilla, will take the best of both familiar world, to carry you to a brand new experience! Because you know <em>Les Bonnes Choses</em> always finds out just how to renew your classics for you...</p>
    </li>
    <li>
        <h1>Cool Coconut Macaron</h1>
        <p>If you ever met coconut taste on its bad day, you surely know that coconut, coming from bad-tempered islands, can be rough sometimes. That is why we like to soften it with a touch of caramel taste in its ganache. The result is the perfect encounter between the finest palm fruit and the most tasty of sugarcane's offspring.</p>
    </li>
    <li>
        <h1>Dark Chocolate Macaron</h1>
        <p>There is so much rich, decadent chocolate in this <strong>macaron</strong> that the chocolatey pleasure is just endless! </p><p>As you let the softness and the strength of our preciously selected cocoa waken your senses, you will feel a lot of pleasure and a little bit of guilt, for giving in to your dark (chocolate) side again!</p><p>There's no mistery why our Dark Chocolate Macaron is our longest-running pastry, in our whole collection, since our creation in 1992.</p>
    </li>
    <li>
        <h1>Pistachio Macaron</h1>
        <p>If you are a <em>nut-cuisine-savvy gourmet</em>, you probably noticed that industrial pistachio flavour tastes very different than actual pistachio. </p><p>And yet, <strong>pistachio nuts shelter a pure, delicate taste, that needs preserving as much as possible</strong>, until it lands in your hands. Therefore, as you bite into our pistachio macaron, you will feel the pistachio nut bits not only delicately set on top of the macaron, but also inside the ganache itself.</p>
    </li>
    <li>
        <h1>Salted Caramel Macaron</h1>
        <p>It's no wonder why our "Salted Caramel French Macaron" has become our best-selling macaron: on top of the authentic Parisian preparation that has been making our macarons so popular and enjoyed by gourmets, you can also feel the waves of Britanny's ocean as you bite into it. Two of the best French local gastronomies meet in your mouth, and suddenly, it's like French Revolution all over again, contained in a single macaron.</p>
    </li>
    <li>
        <h1>Speculoos Macaron</h1>
        <p>Did you know that World War I warriors on the French-Belgian frontier called an unauthorized truce in December 1914, so they could celebrate Christmas together? That's because in every situation, there's much joy to create and receive, from the natural friendship between the French and the Belgian cultures! So when Belgian speculoos meets French macarons, you get to experience the natural chemistry of cultures that are meant to work perfectly together...</p>
    </li>
    <li>
        <h1>Vanilla Macaron</h1>
        <p>Experience the ultimate vanilla experience. Our vanilla Macarons are made with our very own (in-house) <strong>pure extract of Madagascar vanilla</strong>, and subtly dusted with <strong>our own vanilla sugar</strong> (which we make from real vanilla beans).</p>
    </li>
</ul>
<ul>
    <li>
        <h1>Art Director</h1>
        <a href="/store/UlfoxUnM0wkXYXbb/paris-saint-lazare">/store/UlfoxUnM0wkXYXbb/paris-saint-lazare</a>
        <p>Brand visual identity must be vital to you, and you must have a thorough experience with working for/with great visual brands. We will ask you to present your previous work, and explain your choices, so be prepared. Previous work with luxury brands is a plus.</p>
        
    </li>
    <li>
        <h1>Store Intern</h1>
        <a href="/store/UlfoxUnM0wkXYXbc/new-york-fifth-avenue">/store/UlfoxUnM0wkXYXbc/new-york-fifth-avenue</a>
        <p>We expect only one skill from our store interns: an infinite thirst to learn about fine pastry. You might not know much about how to prepare them. You might not know much about how to sell them, and advise about them. You might not know how a pastry shop runs. We've got all of that covered for you.</p><p>Be sure to have the proper motivation, and your future with us will shine bright: we've been offering jobs to all of our interns at the end of their internship.</p>
        
    </li>
    <li>
        <h1>Content Director</h1>
        <a href="/store/UlfoxUnM0wkXYXbb/paris-saint-lazare">/store/UlfoxUnM0wkXYXbb/paris-saint-lazare</a>
        <p>As a company whose marketing is very content-centric, we expect our Content Director to have a tremendous experience, both in content strategy, and in content writing. We expect our applicants to show off some of the content strategies they set up themselves, explaining their choices, and to provide amazing contents they personally wrote.</p><p>Our contents get flexibly powerfully shared on various supports: our site, our in-store printed magazine, our mobile apps, our mailings ... Our Content Director must have experience with all of those, and with using modern adaptive content managers such as <a href="http://prismic.io">prismic.io</a>.</p>
        
    </li>
    <li>
        <h1>Ganache Specialist</h1>
        <a href="/store/UlfoxUnM0wkXYXbb/paris-saint-lazare">/store/UlfoxUnM0wkXYXbb/paris-saint-lazare</a>
        <p>It takes a solid whipping and crystalizing experience to build perfect ganaches. Ganaches can be built from the top down (building a hard material, and softening it), or from the bottom up (coming from a near-liquid state, and making it thicker), and only you know just the way to go to get the perfect output.</p><p>Experienced and highly motivated in fine pastry, you are able to come up with better techniques and formulas to make your art always better.</p>
        
    </li>
    <li>
        <h1>Community Manager</h1>
        <a href="/store/UlfoxUnM0wkXYXbb/paris-saint-lazare">/store/UlfoxUnM0wkXYXbb/paris-saint-lazare</a>
        <p>Mastering the art of direct communication is an experimented skill by itself; and mastering the ability to adapt to the ever-changing modern communication environment is another one. As our community manager, you will need to be proficient with both.</p><p>You will also need to have experience in both aspects, and it's much better if you have worked for/with luxury brands before (not necessarily food-related).</p>
        
    </li>
    <li>
        <h1>Oven Instrumentist</h1>
        <a href="/store/UlfoxUnM0wkXYXbc/new-york-fifth-avenue">/store/UlfoxUnM0wkXYXbc/new-york-fifth-avenue</a>
        <p>A musical instrument does nothing by itself; a musical instrumentalist is but a person. But together, they play an art that is bigger than the sum of the man and the instrument, and generate powerful emotions. It takes a lot of technique; but it takes a lot more than technique. We only ask you one thing: that you do have that technique, and that extra thing.</p>
        
    </li>
    <li>
        <h1>Pastry Dresser</h1>
        <a href="/store/UlfoxUnM0wkXYXbU/tokyo-roppongi-hills">/store/UlfoxUnM0wkXYXbU/tokyo-roppongi-hills</a>
        <p>If you are the kind of person who gets intense, insane, almost neurotic about perfection and details, you are exactly the type we're looking for; but be aware that this job will not make your condition better! With an art background and a perfect mastery of the finest paintbrushes, you can modify every bit of an object to make it shine even brighter.</p>
        
    </li>
    <li>
        <h1>Store Assistant Manager</h1>
        <a href="/store/UlfoxUnM0wkXYXbP/paris-champ-elysees">/store/UlfoxUnM0wkXYXbP/paris-champ-elysees</a>
        <p>To be straightforward: as a Store Assistant Manager, your skills will be as strong as the Store Manager, meaning you can manage your teams, balance the books, give any kind of advice to your customers, ... simply run the store! You need to have extensive experience with pastry (or at least, fine cuisine), and to be customer-service-driven.</p><p>The Store Manager is usually a more seasoned <em>Les Bonnes Choses</em> leader, who already had the time to adapt his leadership style to the <em>Les Bonnes Choses</em> approach.</p>
        
    </li>
    <li>
        <h1>Fruit Expert</h1>
        <a href="/store/UlfoxUnM0wkXYXbb/paris-saint-lazare">/store/UlfoxUnM0wkXYXbb/paris-saint-lazare</a>
        <p>Here at Les Bonnes Choses, we acknowledge that it's a talent to know how to turn fresh material by nature, such as fruit, into something that plays well with the other pastry ingredients. Fruit are natural and temperamental, so you need to know exactly how to tame them and convice them to behave as per your will.</p>
        
    </li>
</ul>
</body>
</html>
